.travis.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         - make bin
         - sudo cp odo /usr/bin
         - oc login -u developer
-        - make test-main-e2e
+        - make test-e2e ODO_TEST_SCENARIO="odoe2e"
```

Comment:
Perhaps we should call this `odoMaine2e`?

Reply:
hmm...Looks precise, will update it.
Thanks for the feedback :)

Reply:
yw :)

Reply:
`odoMaine2e` has weird capitalization. I'd rather have `odomaine2e` or `odo-main-e2e` instead.

Reply:
Then `odoMainE2e` like the rest of the changes use?

Reply:
That feels weird as well, but at least, it's consistent :)

Reply:
I don't mind either way